pkgs/by-name/on/onedrive/package.nix

```
@@ -70,7 +70,7 @@ stdenv.mkDerivation (finalAttrs: {
 
     for s in $out/lib/systemd/user/onedrive.service $out/lib/systemd/system/onedrive@.service; do
       substituteInPlace $s \
-        --replace-fail "/usr/bin/sleep" "${coreutils}/bin/sleep"
+        --replace-fail "/bin/sh -c 'sleep'" "${coreutils}/bin/sleep"
```

```suggestion
        --replace-fail "/bin/sh -c 'sleep 15'" "${lib.getExe' coreutils "sleep"} 15"
```

Otherwise the substitute fails, because the closing `'` is after `15` and not after `sleep`.